browser/ui/views/brave_actions/brave_shields_action_view.cc

```diff
@@ -249,6 +243,23 @@ std::u16string BraveShieldsActionView::GetTooltipText(
   return brave_l10n::GetLocalizedResourceUTF16String(IDS_BRAVE_SHIELDS);
 }
 
+void BraveShieldsActionView::OnThemeChanged() {
```

Comment:
Optional: I think having a base type for views in `brave_action/` would be better so that we can extract duplicated routines like this.

Reply:
Yeah, it can be done anytime you want. I think `template` could be useful in that case.